test/response.spec.ts

```diff
@@ -1066,4 +1066,37 @@ test.group('Response', (group) => {
 
     await supertest(server).get('/').expect(200)
   })
+
+  test('abortUnless: abort request when condition is falsy', async (assert) => {
+    const server = createServer((req, res) => {
+      const config = fakeConfig()
+      const response: Response = new Response(req, res, config)
```

Comment:
See https://github.com/microsoft/TypeScript/pull/33622

It is necessary here to explicitly declare the type of `Response`. In practice in Adonis, the type is already defined in the context so the user doesn't have to do anything.

Example with the change, where TS correctly determined that the code after the call is unreachable:
![image](https://user-images.githubusercontent.com/2352663/71176262-c168c200-2269-11ea-8d03-aeb17a0699e9.png)